pkg/odo/cli/list/list.go

```diff
@@ -0,0 +1,317 @@
+package list
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"text/tabwriter"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/kr/pretty"
+	"github.com/mattn/go-runewidth"
+	"github.com/redhat-developer/odo/pkg/component/labels"
+	"github.com/redhat-developer/odo/pkg/devfile"
+	"github.com/redhat-developer/odo/pkg/devfile/location"
+	"github.com/redhat-developer/odo/pkg/machineoutput"
+	"github.com/redhat-developer/odo/pkg/util"
+	"github.com/rodaine/table"
```

Comment:
I have some concerns about this library. It is not used much, and there is only one persons who maintain this :-/ And it hasn't been updated in a year, and in the last 5 years there only 4 comments and 2 of them were non-code related.